### PR TITLE
Run3 updates

### PR DESCRIPTION
**Pull request by mafrahm**

### CODE REVIEW

File: hbw/production/prepare_objects.py [hunk 1]
```diff
@@ -109,25 +112,24 @@ def prepare_objects(self: Producer, events: ak.Array, results: SelectionResult =
...
-        veto_lepton = ak.concatenate([
-            ak.zip({f: events.VetoMuon[f] for f in lepton_fields}),
-            ak.zip({f: events.VetoElectron[f] for f in lepton_fields}),
-        ], axis=-1)
+        veto_lepton = ak.concatenate([events.VetoMuon * 1, events.VetoElectron * 1], axis=-1)
+        # lepton_fields = set(events.VetoMuon.fields).intersection(events.VetoElectron.fields)
+        # veto_lepton = ak.concatenate([
+        #     ak.zip({f: events.VetoMuon[f] for f in lepton_fields}),
+        #     ak.zip({f: events.VetoElectron[f] for f in lepton_fields}),
+        # ], axis=-1)
```
Comment (apaasch) [inline hbw/production/prepare_objects.py:120]: Remove ? Mksaes code unclean

File: hbw/production/prepare_objects.py [hunk 2]
```diff
@@ -109,25 +112,24 @@ def prepare_objects(self: Producer, events: ak.Array, results: SelectionResult =
...
-        lepton = ak.concatenate([
-            ak.zip({f: events.Muon[f] for f in lepton_fields}),
-            ak.zip({f: events.Electron[f] for f in lepton_fields}),
-        ], axis=-1)
+        lepton = ak.concatenate([events.Muon * 1, events.Electron * 1], axis=-1)
+        # lepton_fields = set(events.Muon.fields).intersection(events.Electron.fields)
+        # lepton = ak.concatenate([
+        #     ak.zip({f: events.Muon[f] for f in lepton_fields}),
+        #     ak.zip({f: events.Electron[f] for f in lepton_fields}),
+        # ], axis=-1)
```
Comment (apaasch) [inline hbw/production/prepare_objects.py:130]: See above

File: hbw/production/prepare_objects.py (hunk 1 above)
Comment (mafrahm) [reply hbw/production/prepare_objects.py:120]:
```suggestion
```

File: hbw/production/prepare_objects.py (hunk 2 above)
Comment (mafrahm) [reply hbw/production/prepare_objects.py:130]:
```suggestion
```

**Review by apaasch — APPROVED** [review]

**Pull request merged**